Crash with very old cuda installed (6.4)

## Description

Djl is failing when an old cuda is installed (64). It seems to be because djl assumes a cuda version is 3 digits long.

### Expected Behavior

It should not crash and fall back to cpu if needed.

### Error Message

```
ai.djl.util.cuda.CudaUtils -- Found cudart: C:\Program Files (x86)\NVIDIA Corporation\PhysX\Common\cudart64_65.dll
Caused by: java.lang.StringIndexOutOfBoundsException: begin 2, end 5, length 4
        at java.base/java.lang.String.checkBoundsBeginEnd(Unknown Source)
        at java.base/java.lang.String.substring(Unknown Source)
        at ai.djl.util.Platform.matches(Platform.java:300)
        at ai.djl.util.Platform.detectPlatform(Platform.java:81)
        at ai.djl.huggingface.tokenizers.jni.LibUtils.copyJniLibraryFromClasspath(LibUtils.java:82)
        at ai.djl.huggingface.tokenizers.jni.LibUtils.loadLibrary(LibUtils.java:66)
        at ai.djl.huggingface.tokenizers.jni.LibUtils.<clinit>(LibUtils.java:41)
```

## How to Reproduce?

Install cuda with a version with 2 numbers (aka before 10.x like 6.4).

## Environment Info

Djl 0.22.1 on windows

--- a/api/src/main/java/ai/djl/util/cuda/CudaUtils.java
+++ b/api/src/main/java/ai/djl/util/cuda/CudaUtils.java
@@ -23,6 +23,7 @@ import org.slf4j.LoggerFactory;
 
 import java.io.File;
 import java.lang.management.MemoryUsage;
+import java.util.Locale;
 import java.util.regex.Pattern;
 
 /** A class containing CUDA utility methods. */
@@ -100,7 +101,7 @@ public final class CudaUtils {
         int version = getCudaVersion();
         int major = version / 1000;
         int minor = (version / 10) % 10;
-        return String.valueOf(major) + minor;
+        return String.format(Locale.ROOT, "%02d", major) + minor;
     }
 
     /**

